rackcity/views/change_plan_views.py

```diff
@@ -408,7 +410,13 @@ def change_plan_execute(request, id):
     num_modified = 0
     num_decommissioned = 0
     updated_asset_mappings = {}
-    for asset_cp in assets_cp:
+
+    # rackmount assets first
+    rackmount_assets_cp = assets_cp.filter(
+        ~Q(model__model_type=ModelType.BLADE_ASSET.value)
```

Comment:
to fix execute bug, first go through and copy over chassis and rackmount assets so then we can assign the chassis to the blades